src/include/S3/S3TransferManager.h

```diff
@@ -53,6 +53,11 @@
  */
 @property (nonatomic, assign) uint32_t minimumUploadPartSize;
 
+/*
+ * Can be set to NO if multipart uploading should not be used, even if the object size  exceeds minimumUploadPartSize.
+ */
+@property (nonatomic) BOOL allowMultipartUpload;
```

Comment:
My thoughts on this:
I don't think it's necessary, as we could just change our AmazonClientManager in modi to mess with `multipartUploadThreshold` based on whether Classic or Leroy is making the request. (For example, just add a flag to the AmazonClientManager for disabling multipart and then using that flag to change the line at https://github.com/hudl/modi/commit/89446c75f08297b239022964898b5a924cf78f1e dynamically. i.e if allow, change back to 10 MB or something, otherwise leave at 500 MB.

This was just me being super safe. Now we can just toggle a flag on the S3TransferManager class and not worry about what the value of `multipartUploadTheshold` is (so I can change that back to 10 MB for all requests)